app/client/data/group-course.ts

```
@@ -1,18 +1,250 @@
 import { Group } from "../types/group";
 
+const parseDate = (dateStr: string): Date => {
+  const cleanedDateStr = dateStr.replace('at', '').trim();
+  const parsedDate = new Date(cleanedDateStr);
+  return isNaN(parsedDate.getTime()) ? new Date() : parsedDate;
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)